src/reactviews/pages/SchemaDesigner/schemaDesigner.tsx

```diff
@@ -0,0 +1,124 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import { useContext, useEffect } from "react";
+import { SchemaDesignerContext } from "./schemaDesignerStateProvider";
+import * as azdataGraph from "azdataGraph";
+import "azdataGraph/dist/index.css";
+import "azdataGraph/src/css/common.css";
+import "azdataGraph/src/css/explorer.css";
+import "./schemaDesigner.css";
+import {
+    addTableIcon,

```

Comment:
off-topic, but do you like the code formatting like this?  I think our "pretty" linting rules makes the code look like lisp-style or something, like where the column width is so short and it takes a ton of rows?